src/app-layout/index.tsx

```diff
@@ -376,30 +376,6 @@ const OldAppLayout = React.forwardRef(
       ? { minWidth: defaults.minContentWidth, maxWidth: defaults.maxContentWidth }
       : undefined;
 
-    const toolsDrawerWidth = (() => {
-      if (isMobile) {
-        return 0;
-      }
-
-      const toolsPanelWidth = toolsHide ? 0 : toolsOpen ? toolsWidth : closedDrawerWidth;
-      const splitPanelWidth =
-        !splitPanelDisplayed || finalSplitPanePosition !== 'side'
-          ? 0
-          : splitPanelOpen
-          ? splitPanelReportedSize
-          : closedDrawerWidth;
-
-      return toolsPanelWidth + splitPanelWidth;
-    })();
-
-    const navigationDrawerWidth = (() => {
-      if (isMobile) {
-        return 0;
-      }
-
-      return effectiveNavigationWidth;
```

Comment:
Drawer component already takes care of these conditions:

https://github.com/cloudscape-design/components/blob/222b99ff1eab212a62dcf7a5154055623e6d38da/src/app-layout/drawer.tsx#L67-L68